units/systemd-hostnamed.service.in

```diff
@@ -22,7 +22,6 @@ IPAddressDeny=any
 LockPersonality=yes
 MemoryDenyWriteExecute=yes
 NoNewPrivileges=yes
-PrivateDevices=yes
```

Comment:
Can we replace this with `DeviceAllow=/dev/vhost-vsock`?

Reply:
Why is the DeviceAllow= added even if PrivateDevices=yes was removed? Isn't that unneeded now?

Reply:
I replaced PrivateDevices=yes (which overmounts /dev/ with a private tmpfs that only has /dev/null and friends) by DeviceAllow= (which leaves /dev/ as it is from the host, but then puts a restrictive cgroup-enforced BPF allowlist on it).

They use different mechanisms. YOu can use one or the other, but it doesn't make much sense to use both together because you'd have an empty /dev/ you put an extra allowlist on..

Reply:
@bluca `DeviceAllow=` implies that only the specified devices can be accessed and no others can be accessed.

Reply:
ah yes, I always forget about that implicit access control